snap/snapcraft.yaml

```diff
@@ -19,6 +22,13 @@ layout:
 apps:
   mysqlrouter:
     command: usr/bin/mysqlrouter
+  mysqlrouter-service:
+    command: start_mysql_router.sh
```

Comment:
as i understand it, the workflow for how mysqlrouter is used in the charms and will be used elsewhere is:
1. user needs to run `mysqlrouter --bootstrap` with the appropriate database credentials. this will create config files and systemd unit files for the mysqlrouter service (tuned to the database whose credentials were provided)
2. user registers these systemd unit files as a systemd service

im not sure how we can define a daemon that systemd unit files that have yet to be generated, but i think `start_mysql_router.sh` would be a good command for the `mysqlrouter` app since it will the mysqlrouter command with reduced privileges.

happy to discuss more in a meeting as well

Reply:
Discussed offline:
User can run the mysqlrouter binary in the snap to run `mysqlrouter --bootstrap` with the `--directory` flag (the directory being the systemd files for the `mysqlrouter-service` daemon). Then the user can enable the snap daemon service (which will use the bootstrap generated systemd files)

@taurus-forever thoughts on the above?